### PR TITLE
Feature/Analytics

### PR DESCRIPTION
**Pull request by brunoocasali**
- Add `User-Agent` inside the pre-defined headers
 
Add ruby support as requested here meilisearch/integration-guides#150

### CODE REVIEW

File: spec/meilisearch/index/base_spec.rb [hunk 1]
```diff
@@ -107,7 +107,8 @@
     expect(index.options).to eq({
                                   headers: {
                                     'Content-Type' => 'application/json',
-                                    'Authorization' => "Bearer #{MASTER_KEY}"
+                                    'Authorization' => "Bearer #{MASTER_KEY}",
+                                    'User-Agent' => MeiliSearch.qualified_version
```
Comment (curquiza) [inline spec/meilisearch/index/base_spec.rb:111]: Should be `User-agent: Meilisearch (<version>)`

Comment (brunoocasali) [reply spec/meilisearch/index/base_spec.rb:111]: Actually, it should be `User-Agent` https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent.

Or your complaint is about the idea to have a method to do that?

Comment (curquiza) [reply spec/meilisearch/index/base_spec.rb:111]: I'm complaining about the `Meilisearch Ruby (<version>)` where the `Meilisearch Ruby` part is missing, if I'm not wrong

**Review by curquiza — DISMISSED** [review]
💪

File: lib/meilisearch/version.rb [hunk 2]
```diff
@@ -2,4 +2,8 @@
 
 module MeiliSearch
   VERSION = '0.18.0'
+
+  def self.qualified_version
+    "MeiliSearch Ruby (v#{VERSION})"
```
Comment (curquiza) [inline lib/meilisearch/version.rb:7]:
```suggestion
    "Meilisearch Ruby (v#{VERSION})"
```

**Comment by brunoocasali** [comment]
bors merge

**Review by curquiza — APPROVED** [review]
👌

**Comment by brunoocasali** [comment]
bors try

**Review by bidoubiwa — APPROVED** [review]
LGTM :fire: :fire:

**Comment by brunoocasali** [comment]
bors merge

**Pull request merged**